### PR TITLE
[4.0] Fix codemiror, use original css files

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #27832 .

### Summary of Changes

I have changed codemiror layout to use original css files. 

### Testing Instructions
Pleas look for original issue #27832

### Expected result
codemiror works

### Actual result
codemiror does not works

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e973144314fa678651b354b31ba2e9b306364153<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27847">issues.joomla.org/tracker/joomla-cms/27847</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on e973144314fa678651b354b31ba2e9b306364153<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27847">issues.joomla.org/tracker/joomla-cms/27847</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27847">issues.joomla.org/tracker/joomla-cms/27847</a>.</sub>

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php [hunk 1]
```diff
@@ -36,8 +35,8 @@
 $addons          = 'addons="' . ltrim(HTMLHelper::_('script', $basePath . 'lib/addons' . $extJS, ['version' => 'auto', 'pathOnly' => true]), '/') . '"';
 
 Factory::getDocument()->getWebAssetManager()
-	->registerAndUseStyle('codemirror.lib.main', $basePath . 'lib/codemirror' . $extCSS)
-	->registerAndUseStyle('codemirror.lib.addons', $basePath . 'lib/addons' . $extCSS, [], [], ['codemirror.lib.main'])
+	->registerAndUseStyle('codemirror.lib.main', $basePath . 'lib/codemirror.css' )
```
Comment (Quy) [inline plugins/editors/codemirror/layouts/editors/codemirror/element.php]:
```suggestion
	->registerAndUseStyle('codemirror.lib.main', $basePath . 'lib/codemirror.css')
```

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**